samples/nios2-unknown-elf/crosstool.config

```diff
@@ -0,0 +1,5 @@
+CT_ARCH_nios2=y
+CT_CC_LANG_CXX=y
+CT_DEBUG_gdb=y
+# CT_GDB_CROSS_PYTHON is not set
+CT_GETTEXT=y
```

Comment:
Hmm... Gettext is definitely a leftover from a canadian sample, that I used as an example.